src/aliases/manager/service.go

```diff
@@ -1,50 +1,19 @@
 package aliasmanager
 
 import (
-	"context"
-
-	aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
-	aliasstore "github.com/consensys/quorum-key-manager/src/aliases/store"
+	"github.com/consensys/quorum-key-manager/pkg/app"
+	aliaspg "github.com/consensys/quorum-key-manager/src/aliases/store/postgres"
+	"github.com/consensys/quorum-key-manager/src/infra/postgres"
 )
 
-type BaseManager struct {
-	db aliasstore.Database
-}
-
-func New(db aliasstore.Database) *BaseManager {
-	return &BaseManager{
-		db: db,
+// RegisterService creates and register the alias service in the app.
+func RegisterService(a *app.App, pgClient postgres.Client) error {
```

Comment:
Following conventions from `nodes` and `stores`, this code belongs to `/src/aliases/service.go`

Reply:
In stores, that code is in `/src/stores/app/service.go`. For `nodes` it is directly in the main directory.
I'll do it like that.

Reply:
done